core/object/script_language.h

```diff
@@ -310,6 +310,7 @@ class ScriptLanguage {
 		Ref<Script> script;
 		String class_name;
 		String class_member;
+		String class_path;
```

Comment:
Is it intended that the `master` branch has this addition but the matching `3.x` PR didn't seem to need it? #48616

That's the only reason I hadn't merged this yet as I was waiting for @vnen to check it since changing a `core` struct that affects all language bindings increases the scope of this PR significantly. On the other hand I see that `LookupResult` only seems to be used by the GDScript editor so it's probably fine.

Reply:
Yes, the fundamentally there were different values between GDScript 2.0 and GDScript 1.0 that made accessing the base class for an extending class impossible in Godot 4.0, so I needed a way to reach it at the end of the day.

I made sure to only add a value, and changed nothing else about the code, so that it won't break any bindings that don't need it.